packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1826,6 +1828,22 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
         _cursorTimer = null;
       }
     }
+
```

Comment:
Nit: There's no need to do any of the below if the platform isn't android or iOS.  You could return early here on those other platforms, but I'm not sure how much benefit you would get from that in reality.  Up to you @markusaksli-nc if you want to do it.

Reply:
I thought this would make the purpose and value of `_lastOrientation` a bit niche and opaque but now that I think about it, it wouldn't reflect physical rotation on other platforms anyway.